_opensearch/cluster.md

```diff
@@ -12,23 +12,26 @@ OpenSearch can operate as a single-node or multi-node cluster. The steps to conf
+### Nodes
 
 The following table provides brief descriptions of the node types:
 
 Node type | Description | Best practices for production
 :--- | :--- | :-- |
-`Master` | Manages the overall operation of a cluster and keeps track of the cluster state. This includes creating and deleting indices, keeping track of the nodes that join and leave the cluster, checking the health of each node in the cluster (by running ping requests), and allocating shards to nodes. | Three dedicated master nodes in three different zones is the right approach for almost all production use cases. This configuration ensures your cluster never loses quorum. Two nodes will be idle for most of the time except when one node goes down or needs some maintenance.
-`Master-eligible` | Elects one node among them as the master node through a voting process. | For production clusters, make sure you have dedicated master nodes. The way to achieve a dedicated node type is to mark all other node types as false. In this case, you have to mark all the other nodes as not master-eligible.
-`Data` | Stores and searches data. Performs all data-related operations (indexing, searching, aggregating) on local shards. These are the worker nodes of your cluster and need more disk space than any other node type. | As you add data nodes, keep them balanced between zones. For example, if you have three zones, add data nodes in multiples of three, one for each zone. We recommend using storage and RAM-heavy nodes.
-`Ingest` | Preprocesses data before storing it in the cluster. Runs an ingest pipeline that transforms your data before adding it to an index. | If you plan to ingest a lot of data and run complex ingest pipelines, we recommend you use dedicated ingest nodes. You can also optionally offload your indexing from the data nodes so that your data nodes are used exclusively for searching and aggregating.
-`Coordinating` | Delegates client requests to the shards on the data nodes, collects and aggregates the results into one final result, and sends this result back to the client. | A couple of dedicated coordinating-only nodes is appropriate to prevent bottlenecks for search-heavy workloads. We recommend using CPUs with as many cores as you can.
+Cluster manager | Manages the overall operation of a cluster and keeps track of the cluster state. This includes creating and deleting indexes, keeping track of the nodes that join and leave the cluster, checking the health of each node in the cluster (by running ping requests), and allocating shards to nodes. | Three dedicated `cluster_manager` nodes in three different zones is the right approach for almost all production use cases. This configuration ensures your cluster never loses quorum. Two nodes will be idle for most of the time except when one node goes down or needs some maintenance.
```

Comment:
Yes. we are using the code font when we refer to the actual node. Let me remove the code font here.

We don't need a hyphen, because cluster is the adjective for the noun "manager". You only need a hyphen to separate two adjectives that modify a noun.

Reply:
Hi @alicejw-aws Thanks for making the change and sharing the usage of hyphen! Got it. 👍

Reply:
Thanks @tlfeng for verifying the changes, do you think there are any other instances that we missed, or is it ready to go for 2.0?

Reply:
The changes are all correct, so this Pull Request is ready to go for version 2.0.
But there might be some other items regarding deprecating "master" term. 
My concern is that the current "master" usages in Rest API and Setting are not removed in version 2.0. Is it fine to remove all the "master" usages in documentation? Should add them back in some time ?

Reply:
ok thanks @tlfeng. 

I will keep this PR open so we can add the  list of deprecated REST APIs and settings you mentioned in  the 450 overview:
( https://github.com/opensearch-project/OpenSearch/issues/1549)